ddtrace/contrib/botocore/patch.py

```diff
@@ -190,13 +202,14 @@ def get_kinesis_data_object(data):
 
     # check if data is a json string
     try:
-        return json.loads(data)
+        return get_json_from_str(data)
     except ValueError:
         pass
 
     # check if data is a base64 encoded json string
     try:
-        return json.loads(base64.b64decode(data).decode("ascii"))
+        data_str = base64.b64decode(data).decode("ascii")
+        return get_json_from_str(data_str)
```

Comment:
Is there a reason to break out the logic in this function if we don't test it specifically and don't reuse it?

Reply:
ooops nvm, I see we try it twice.